crates/apollo-parser/src/parser/grammar/operation.rs

```diff
@@ -1,46 +1,6 @@
 use crate::parser::grammar::value::Constness;
-use crate::parser::grammar::{directive, name, selection, ty, variable};
-use crate::{Parser, SyntaxKind, TokenKind, S, T};
-
-/// RootOperationTypeDefinition is used in a SchemaDefinition. Not to be confused
-/// with OperationDefinition.
-///
-/// See: https://spec.graphql.org/October2021/#RootOperationTypeDefinition
-///
-/// *RootOperationTypeDefinition*:
-///    OperationType **:** NamedType
-pub(crate) fn root_operation_type_definition(p: &mut Parser, is_operation_type: bool) {
```

Comment:
this belongs with the schema definition, not with operation definitions, in my opinion